zerver/migrations/0604_set_default_value_for_can_add_custom_emoji_group.py

```diff
@@ -0,0 +1,43 @@
+# Generated by Django 4.2.1 on 2023-06-12 10:47
+
+from django.db import migrations
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+from django.db.models import OuterRef
+
+
+def set_default_value_for_can_add_custom_emoji_group(
+    apps: StateApps, schema_editor: BaseDatabaseSchemaEditor
+) -> None:
+    Realm = apps.get_model("zerver", "Realm")
+    NamedUserGroup = apps.get_model("zerver", "NamedUserGroup")
+
+    add_custom_emoji_policy_to_user_group = {
+        1: "role:members",
+        2: "role:administrators",
+        3: "role:fullmembers",
+        4: "role:moderators",
+    }
+
+    for id, user_group in add_custom_emoji_policy_to_user_group.items():
```

Comment:
s/user_group/group_name/ here would be more clear.

Reply:
I have changed it now, but `user_group` was also used in previous migrations. Should we change that too?

Reply:
Probably not important to change for older ones; it's just for readability and folks copy/paste off the latest similar one usually.